test-data/unit/python2eval.test

```diff
@@ -152,13 +156,16 @@ f(**params)
 
 [case testFromFutureImportUnicodeLiterals2_python2]
 from __future__ import unicode_literals
-def f(x: str) -> None: pass
+def f(x):
+    # type: (str) -> None
+    pass
 f(b'')
 f(u'')
 f('')
 [out]
-_program.py:4: error: Argument 1 to "f" has incompatible type "unicode"; expected "str"
-_program.py:5: error: Argument 1 to "f" has incompatible type "unicode"; expected "str"
+_program.py:5: error: Argument 1 to "f" has incompatible type "bytes"; expected "str"
+_program.py:6: error: Argument 1 to "f" has incompatible type "unicode"; expected "str"
+_program.py:7: error: Argument 1 to "f" has incompatible type "unicode"; expected "str"
```

Comment:
Note that all three calls are now in error! bytes is not compatible with str, and `''` becomes unicode because of the `unicode_literals`.